tests/framework/extensions/kubeapi/projects/create.go

```diff
@@ -0,0 +1,33 @@
+package projects
+
+import (
+	"context"
+
+	"github.com/rancher/rancher/pkg/api/scheme"
+	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/extensions/unstructured"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// CreateProject is a helper function that uses the dynamic client to create a project.
+func CreateProject(client *rancher.Client, project *v3.Project) (*v3.Project, error) {
+	dynamicClient, err := client.GetDownStreamClusterClient(localcluster)
```

Comment:
Shouldn't this be downstream cluster ID instead of `localcluster`?

Reply:
Yes it should allow both local and downstream. I will modify the helper functions that I created in this PR for Projects.

Reply:
Project CRDs are available only on the local cluster and not on the downstream cluster. To create a project on the downstream cluster, we need to include the cluster ID of the downstream cluster for namespace in the project yaml. Hence, we pass Namespace here: dynamicClient.Resource(ProjectGroupVersionResource).**Namespace(project.Namespace)**